### PR TITLE
feat: body size for non-intercepted requests

### PR DESCRIPTION
**Pull request by Lightning00Blade**
I couldn't get an example with multiple `postDataEntries` to verify that we can just add things like that.

### CODE REVIEW

File: wpt-metadata/chromedriver/headful/webdriver/tests/bidi/input/set_files/invalid.py.ini [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[invalid.py]
+  expected: TIMEOUT
```
Comment (sadym-chromium) [inline wpt-metadata/chromedriver/headful/webdriver/tests/bidi/input/set_files/invalid.py.ini:2]: This one seems to be irrelevant

**Review by sadym-chromium — APPROVED** [review]

File: wpt-metadata/chromedriver/headful/webdriver/tests/bidi/input/set_files/invalid.py.ini (hunk 1 above)
Comment (Lightning00Blade) [reply wpt-metadata/chromedriver/headful/webdriver/tests/bidi/input/set_files/invalid.py.ini:2]: Auto-updated. Will check one test is failing unexpectedly
